support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -7,6 +7,9 @@ import { V1 } from 'helpers/abTests/data/testAmountsData';
 // ----- Tests ----- //
 export type LandingPageCopyReturningSinglesTestVariants = 'control' | 'returningSingle' | 'notintest';
 export type LandingPageMomentBackgroundColourTestVariants = 'control' | 'yellow' | 'notintest';
+export type LandingPageStripeElementsRecurringTestVariants = 'control' | 'stripeElements' | 'notintest';
+
+const contributionsLandingPageMatch = '/(uk|us|eu|au|ca|nz|int)/contribute(/.*)?$';
```

Comment:
The `targetPage` field in the `Test` object is a new feature for restricting tests to specific pages on support.thegaurdian.com. I will apply this matcher to our other landing page tests, but not in this PR